public/src/css/style.css

```diff
@@ -34,6 +34,33 @@ a:hover,
     color: #2ea3eb;
 }
 
+.collections-published {
+    background-color: #fff;
+    padding: 5px 10px;
+    box-shadow: 0 1px 6px rgba(0, 0, 0, 0.32), 0 1px 6px rgba(0, 0, 0, 0.32);
```

Comment:
`box-shadow` and few other properties are already defined in `cardContainer` we can avoid duplication